### ISSUE TITLE
Приложения телеграма не работают, но бот через API публикует посты

### ISSUE THREAD

**Issue by iprst** [terratensor/tg-svodd-bot#24]
Просто наблюдение. 

```
8 июня, в популярном мессенджере Telegram произошел массовый сбой. Примерно с 21:25 по московскому времени возникли проблемы, из-за которых Telegram перестал загружаться. Причина возникшей проблемы на данный момент неизвестна.
В поисковиках фиксируется огромное количество запросов на эту тему. Telegram не работает ни с телефонов, ни с компьютеров.
```

**Comment by audetv** [comment]
У меня вроде открывается телеграмм, но была 40 мин назад зарегистрирована ошибка, пришло сообщение:
```
error: failed to send successful request. Status was "429 Too Many Requests". Response body: map[description:Too Many Requests: retry after 365 error_code:429 ok:false parameters:map[retry_after:365]] Text: 
<i>Думаю, пригодится.</i>

И практически паралельно была новость, что астронавт Апполо8 разбился. Тот который якоыб на Луну летал в 68. Какая то связь бесструктурно явно есть
90-летний участник «Аполлона-8» разбился на самолете
Участник экипажа «Аполлон-8» Уильям Андерс погиб в пятницу, 7 июня. Самолет, который 90-летний космонавт пилотировал в одиночку, упал в воду у островов Сан-Хуан (штат Вашингтон).
https://www.kommersant.ru/doc/6760308

★ <a href="https://xn----8sba0bbi0cdm.xn--p1ai/qa/question/view-30604#:~:text=19:22 08.06.2024">Источник</a>
```
Т.е. на данный момент последний пост не опубликован, 
Понаблюдаем.

**Comment by audetv** [comment]
Удалили этот пост из БД, чтобы система его заново спарсила и отправила на публикацию. 
Ошибки не наблюдаю. Пост 473265 опубликован.

**Comment by iprst** [comment]
Сейчас телега уже открывается, примерно 40 минут назад восстановилась работоспособность, то есть в 23 МСК. Новый пост появился. Два или три предыдущих поста были опубликованы после глобального падения, я его как раз застал, и видел появление новых постов, но потом видимо и до API докатилась волна, либо это был уже управляемый процесс отключения и перезагрузки серверов или что там сисадмины делают с железом.

**Comment by audetv** [comment]
> Потом либо до API докатилась волна, либо это был уже управляемый процесс отключения и перезагрузки серверов или что там сисадмины делают с железом.

Да, похоже. Я все пропустил) Ошибка не такая как обычно приходит, обычно connection reset by peer Text: или timeout, а в этот раз 429 Too Many Requests, как будто бы сервис получил превышение запросов от пользователя и выбросил ошибку.

> Сейчас телега уже открывается, примерно 40 минут назад восстановилась работоспособность, то есть в 23 МСК. 

Да, я уже после 23:00 залез в инет почитал сообщения, что телеграм сбоит и т.п. Ну, интересно с чем связано, можно подогадываться. А еще видел сообщение, про какую то валюту star от телегарм, которую представил Дуров. может и с этим барахлом

**Comment by iprst** [comment]
> Ошибка не такая как обычно приходит, обычно connection reset by peer Text: или timeout, а в этот раз 429 Too Many Requests, как будто бы сервис получил превышение запросов от пользователя и выбросил ошибку.

Ддос атака была вероятно? В приложениях выглядело просто — пустота, чатов-каналов нет, личную информацию видно, но разлогинится, залогинится, перелогинится или что-то изменить невозможно.

**Comment by audetv** [comment]
Возможно ддос атака, не исключено. Но вряд ли это будет представлено общественности да и не зачем.
Но что интересно, да же в такой казалось бы технической ситуации, некоторыми «военкурами» по смыслу жизни, так сказать, а не по профессии, имеющие мировоззрение военкуровское — все плохо, а хорошо у хозяина. Так вот, не углубляюсь в поиск фактологии, попадал на посты, что это все магистральный Российский провайдер, неважно какой, а у телеграма все ок, ха-ха ничего не умеют, опять сами себя заблокировали. Вот так любой информационный повод используется чтобы уколоть и выразить свою ненависть ко всему русскому.

**Comment by audetv** [comment]
> Возможно ддос атака, не исключено. Но вряд ли это будет представлено общественности да и не зачем. Но что интересно, да же в такой казалось бы технической ситуации, некоторыми «военкурами» по смыслу жизни, так сказать, а не по профессии, имеющие мировоззрение военкуровское — все плохо, а хорошо у хозяина. Так вот, не углубляюсь в поиск фактологии, попадал на посты, что это все магистральный Российский провайдер, неважно какой, а у телеграма все ок, ха-ха ничего не умеют, опять сами себя заблокировали. Вот так любой информационный повод используется, чтобы уколоть и выразить свою ненависть ко всему русскому.

Хотя казалось бы причем здесь телеграм…
